connect/runtime/src/main/java/org/apache/kafka/connect/runtime/AbstractHerder.java

```diff
@@ -151,6 +151,11 @@ protected void stopServices() {
         this.configBackingStore.stop();
         this.worker.stop();
         this.connectorExecutor.shutdown();
+        try {
+            this.connectorClientConfigOverridePolicy.close();
+        } catch (Exception e) {
+            log.warn("Exception while stop connectorClientConfigOverridePolicy:", e);
+        }
```

Comment:
This can be a one-liner with the `Utils` class:

```suggestion
        Utils.closeQuietly(this.connectorClientConfigOverridePolicy, "connector client config override policy");
```